cmd/modelcmd/controller.go

```diff
@@ -104,6 +104,12 @@ func (c *ControllerCommandBase) initController() error {
 }
 
 // DetermineCurrentController returns current controller on this client.
+// The returned result corresponds to, in order of precedence:
```

Comment:
This has been removed from all 2.5 and develop versions.

Reply:
thanks for the heads up, will fix in the next merge if this lands as is.

Reply:
let me know if https://github.com/juju/juju/pull/9821 doesn't address your concerns.